### PR TITLE
[JG] OPS-12383 - changing to retrieve the clearing date

### PR DESCRIPTION
**Pull request by DreamcastKnight**

### CODE REVIEW

File: test/pages/tests/ViewProgressSpec.scala [hunk 1]
```diff
@@ -566,7 +566,7 @@ class ViewProgressSpec extends BrowserSpec {
     ViewProgress.getProgressTimelineItems shouldBe List(
       ProgressTimelineItem(
         "Repayment complete",
-        "24 Oct 2019",
+        "01 Mar 2018",
```
Comment (AndyHWChung) [inline test/pages/tests/ViewProgressSpec.scala:569]: where does the 1st March 2018 date come from in these tests? Also, do we have separate tests to test that the old date is used if there is no clearing date and that the clearing date is used if it's there?

Comment (DreamcastKnight) [reply test/pages/tests/ViewProgressSpec.scala:569]: from what I can see, the 1st march is the clearing date it gets

Comment (AndyHWChung) [reply test/pages/tests/ViewProgressSpec.scala:569]: it seemed a bit odd to me cos the clearing date didn't exist before and the only new clearing date I can see you put into this commit is `2001-01-01` in `DesData` unless I'm missing something 🤔

File: test/pages/tests/ViewProgressSpec.scala [hunk 2]
```diff
@@ -244,7 +244,7 @@ class ViewProgressSpec extends BrowserSpec {
     ViewProgress.getProgressTimelineItems shouldBe List(
       ProgressTimelineItem(
         "Repayment complete",
-        formattedTodayString,
+        "01 Mar 2018",
```
Comment (AndyHWChung) [inline test/pages/tests/ViewProgressSpec.scala:247]: are there unit tests testing what happens if the clearing date is not there?

**Review by AndyHWChung — APPROVED** [review]

**Pull request merged**